src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Repositories/DelegationMetadataRepository.cs

```diff
@@ -40,31 +40,34 @@ public class DelegationMetadataRepository : IDelegationMetadataRepository
             _connectionString = string.Format(
                 postgresSettings.Value.ConnectionString,
                 postgresSettings.Value.AuthorizationDbPwd);
+            NpgsqlConnection.GlobalTypeMapper.MapEnum<DelegationChangeType>("delegation.delegationchangetype");
         }
 
         /// <inheritdoc/>
-        public async Task<bool> InsertDelegation(string altinnAppId, int offeredByPartyId, int? coveredByPartyId, int? coveredByUserId, int delegatedByUserId, string blobStoragePolicyPath, string blobStorageVersionId, bool isDeleted = false)
+        public async Task<DelegationChange> InsertDelegation(DelegationChange delegationChange)
         {
             try
             {
                 using NpgsqlConnection conn = new NpgsqlConnection(_connectionString);
```

Comment:
```suggestion
                await using NpgsqlConnection conn = new NpgsqlConnection(_connectionString);
```

Ser VS anbefaler `await using` her. Ser denne implementerer `IAsyncDisposable` og har en `DisposeAsync` som kaller `CloseAsync`. Uten `await` her kalles `Dispose` og `Close`. Usikker på om det har noen praktisk betydning.

Reply:
fixed this for all operations